lib/splitio-api-binding.js

```diff
@@ -55,40 +54,16 @@ class SplitioApiBinding {
     }
   }
 
-  /**
-   * Polls the Split.io API until since and till timestamps are the same.
-   * @param {String} path - The path of the HTTP request.
-   * @param {Integer} maxNumRequests - The maximum number of API requests to make
-   */
-  async getAllChanges ({ path, maxNumRequests = DEFAULT_MAX_NUM_REQUESTS }) {
-    let since = SINCE_VALUE_FOR_FIRST_REQUEST
-    let requestCount = 0
-    const allChanges = []
-    while (requestCount < maxNumRequests) {
-      const results = await this.httpGet({ path, since })
-      if (since === results.till) {
-        break
-      }
-      allChanges.push(results)
-      since = results.till
-      requestCount++
-    }
-    return { allChanges, since }
-  }
-
   /**
    * Get split data.
    */
   async getSplitChanges () {
     const path = '/splitChanges'
-    const splitChanges = []
-    const { allChanges, since } = await this.getAllChanges({ path })
-    allChanges.forEach(change => {
-      change.splits.forEach(split => {
-        splitChanges.push(split)
-      })
+    const { splits, since } = await this.httpGet({
```

Comment:
Does the API guarantee to return all the splits?  Even if there response is very large?

Should you rename this function -- `getSplitChanges` -> `getSplits`?

Reply:
function rename makes sense.

the doc makes it sound like the API does not guarantee all splits are returned, but after looking into it more, i don't think updating the `since` value is supposed to function as pagination (i've reached out for clarification)